### PR TITLE
Update README with conda info, include LICENSE in tarball

### LINKED ISSUE #202: Upload to Conda
### Goal

Expose Qt.py to users of [Conda](https://conda.io)

### Implementation

See here: https://conda.io/docs/build_tutorials/pkgs.html

### PR DESCRIPTION
**Pull request by fredrikaverpil**
Fixes #202 

Qt.py was today added to conda/Anaconda Cloud/conda-forge and its feedstock lives [here](https://github.com/conda-forge/qt.py-feedstock/blob/master/README.md).

I added the Anaconda Cloud badge to the README and extended the install instructions with a conda example.

What's particularly cool about this, is that PySide2 can be – as of yesterday – _also_ installed with conda:
This makes me very happy:

```bash
$ conda config --add channels conda-forge
$ conda install qt.py pyside2
```

We should make our `LICENSE` be included in the [Qt.py tarball](https://pypi.org/project/Qt.py/#files) instead of keeping a copy of it in [qt.py-feedstock](https://github.com/conda-forge/qt.py-feedstock). I've added `MANIFEST.in` to this PR, which is [supposed to handle this](https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute) but I'm not sure how to test this. We need to trigger a PyPi release (through Travis) to know for sure?

### CODE REVIEW

**Comment by mottosso** [comment]
> I've added MANIFEST.in to this PR, which is supposed to handle this but I'm not sure how to test this. We need to trigger a PyPi release (through Travis) to know for sure?

That is one way of doing it, and you can test it via `python setup.py sdist` *I think*.

Alternatively, there is a `data_files` argument to `setup.py` you could use to include it, then the above will work for sure and you can inspect the resulting zip file.

**Comment by fredrikaverpil** [comment]
> Alternatively, there is a data_files argument to setup.py

I was thinking of that too. But wouldn't that add `LICENSE` to the wheel and cause it to get dropped inside of end-users `site-packages` directory?

**Comment by fredrikaverpil** [comment]
> python setup.py sdist

That worked!

And `LICENSE` is inside.

**Comment by fredrikaverpil** [comment]
Before `MANIFEST.in`: 

screen shot 2017-08-29 at 15 53 46

And after:

screen shot 2017-08-29 at 15 54 36

**Comment by fredrikaverpil** [comment]
I just ran `python setup.py bdist_wheel` to check the contents of the wheel. And with `MANIFEST.in` in place, there are no additional files added to the wheel (such as the license).

**Comment by fredrikaverpil** [comment]
Ok, I just also used `data_files` in `setup.py` and that resulted in this wheel:

screen shot 2017-08-29 at 16 00 04


...and this tarball:

screen shot 2017-08-29 at 16 00 10

I think that's cleaner. So, I'll vote for adding data_files to setup.py and remove MANIFEST.in.

@mottosso what would you prefer?

**Comment by fredrikaverpil** [comment]
Okay, I'm ready if you are. Pull the trigger on this PR if you're happy!

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     url="https://github.com/mottosso/Qt",
     license="MIT",
     zip_safe=False,
+    data_files=['LICENSE'],
```
Comment (mottosso) [inline setup.py]: Nitpick, please use consistent quotation marks `"` :)

Other than that, all good!

Comment (fredrikaverpil) [reply setup.py]: Whoops. Good catch. Fixed.

**Comment by mottosso** [comment]
Cool, win. :)

**Pull request merged**